lib/ansible/modules/system/authorized_key.py

```diff
@@ -91,6 +91,14 @@
     default: "yes"
     choices: ["yes", "no"]
     version_added: "2.1"
+  comment:
+    description:
+      - Change the comment on the public key. Rewriting the comment is useful in
+        cases such as fetching it from Github or Gitlab.
+      - If no comment is specified, the existing comment will be kept.
+    required: false
+    default: null
```

Comment:
You want to state default: None instead of default: null for consistency. Otherwise, LGTM.